json/README.md

```diff
@@ -49,4 +49,28 @@ need it you're better off sticking to the standard library, or spend a bit of
 time implementing it in here ;)
 
 Note that none of those features should result in performance degradations if
-they were implemented in the package.
+they were implemented in the package, and we welcome contributions!
+
+## Trade-offs
+
+As one would expect, we had to make a couple of trade-offs to achieve greater
+performance than the standard library, but there were also features that we
+did not want to give away.
+
+Other open-source packages offering a reduced CPU and memory footprint usually
+do so by designing a different API, or require code generation (therefore adding
+complexity to the build process). These were not acceptable conditions for us,
+as we were not willing to trade off developer productivity for better runtime
+performance. The safest way to achieve this was to replicate exactly standard
```

Comment:
"safest" is an touchy word choice to use here because of the following description of our use of unsafe :)

maybe

"To achieve this, we decided to exactly replicate the standard library ..."

?

Reply:
How about

> To preserve the interface, the package implementation was the only area that we were able to work with.

?